combinations/hash.tsv

```diff
@@ -493,3 +493,4 @@ gatk4=4.4.0.0,samtools=1.18,bcftools=1.18,perl=5.32.1,r-base=4.3
 r-base=4.2.1,r-optparse
 biopython=1.78,pybedtools=0.9.1
 xengsort=2.0.1,pigz=2.8,htop=3.2.2,seqkit=2.6.0
+t1k=1.0.4,samtools=1.18,bcftools=1.18,bbmap=39.01,perl=5.32.1,pigz=2.8,htop=3.2.2,seqkit=2.6.0,csvtk=0.28.0
```

Comment:
htop is probably not needed
Isn't that container not too big in general? Could you run your task in two steps with two containers?